ocamlformat-help.txt

```diff
@@ -183,6 +183,10 @@ OPTIONS (CODE FORMATTING STYLE)
            will always break the line before a single case. The default value
            is compact.
 
+       --space-around-collection-expressions
+           Add a space around collection expressions (lists, arrays,
+           records).
```

Comment:
When I looked at the code, I realized that my first interpretation of this doc string was confused (I expected the added space outside the delimiters, not inside). What about something like: "Add a space inside the delimiters of collection expressions..."?